Dockerfile

```diff
@@ -0,0 +1,62 @@
+# Builder image
+FROM ubuntu:18.04 as builder
+
+# Install build tools and libraries
+RUN dpkg --add-architecture i386 &&\
```

Comment:
The Docker docs recommend against running `apt-get upgrade` or `apt-get dist-upgrade`, "as many of the 'essential' packages from the parent images cannot upgrade inside an unprivileged container". https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

They also state in this section that "Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required".

So I'm hoping these lines can forego the `upgrade`, `autoremove`, and `clean` bits to make the script a bit less verbose. (And then only need to add i386 arch, update package list, and install the needed packages)

This should be the same for the apt-get invocations further in the file.

Reply:
Thanks for the feedback, I didn't know about these Debian/Ubuntu best practices :-)
I just commited/tested/pushed the changes.